app/controllers/subscriptions_controller.rb

```diff
@@ -102,7 +102,12 @@ def email
     params.permit(:email)
   end
 
+  def nqt_job_alert?
+    request.original_fullpath == "/sign-up-for-NQT-job-alerts"
```

Comment:
Is it possible to get this string programmatically, e.g. is it `nqt_job_alerts_url`?

Reply:
Not sure this logic deserves to be its own method - could live inside `origin_param` method as it's simple and not reused.

Reply:
Alternatively you can make this method a `helper_method` so it's available in the view, and then that prevents the string duplication on line 1 of `app/views/shared/search/_keyword.html.slim`.

Reply:
So I'd create a file called `subscriptions_helper.rb` then include that file int the controller so I can use it?

Reply:
If you have a look at the vacancies controller on line 2 it declare a controller method (job application) as a helper method. 

To avoid creating a new file for only one method. I'll copy this in the subscriptions controller and then I can use nqt_job_alert? in the view too. Is that ok with you?

Reply:
Yes - that's the smoothest way to create a helper_method for a specific controller:

```ruby
class ControllerName
  helper_method :my_method
  
  def my_method
    "Jim's the name, being Jim's the game"
  end
end
```